src/SearchResults/index.js

```diff
@@ -21,6 +21,17 @@ var generateHierarchicalTree = require('./generate-hierarchical-tree');
  * @property {object} stats undefined unless facet_stats is retrieved from algolia
  */
 
+ /**
+  * @typedef SearchResults.HierarchicalFacet
+  * @type {object}
+  * @property {string} name name of the current value given the hierarchical level, trimmed.
+  *                         If root node, you get the facet name
+  * @property {integer} count number of objets matching this hierarchical value
```

Comment:
The correct type in JS is number. You can specify the value more precisely in the comment.